core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -3120,19 +3099,16 @@ class KafkaApisTest extends Logging {
   }
 
   @Test
-  def shouldRespondWithUnsupportedMessageFormatForBadPartitionAndNoErrorsForGoodPartition(): Unit = {
-    val tp1 = new TopicPartition("t", 0)
+  def shouldRespondWithNoErrorsForGoodPartition(): Unit = {
```

Comment:
I think we can delete this test. `shouldRespondWithUnknownTopicOrPartitionForBadPartitionAndNoErrorsForGoodPartition` seems to cover this path already.

Reply:
Pushed a commit that does this.